packages/office-addin-project/src/convert.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fs from "fs";
+import { exec } from "child_process";
+import { ExpectedError } from "office-addin-usage-data";
+
+/* global console */
+
+export async function convertProject(manifestPath: string = "./manifest.xml") {
+  if (manifestPath.endsWith(".json")) {
+    throw new ExpectedError(
+      `The convert command only works on xml based projects`
+    );
+  }
+
+  if (!fs.existsSync(manifestPath)) {
+    throw new ExpectedError(
+      `The manifest file '${manifestPath}' does not exist`
+    );
+  }
+
+  updatePackages();
+}
+
+function updatePackages(): void {
+  // Contains name of the package and minimum version
+  const depedentPackages: string[] = [
+    "@microsoft/teamsfx",
+    "office-addin-debugging",
+    "office-addin-manifest",
+  ];
+  let command: string = "npm install";
+  let messageToBePrinted: string = "Installing latest versions of";
+
+  for (let i = 0; i < depedentPackages.length; i++) {
+    const depedentPackage = depedentPackages[i];
+    command += ` ${depedentPackage}@latest`;
```

Comment:
Yeah, I was first doing that way, but then I realized that npm already does that (check the versions and only upgrade if necessary).
So we might as well use what is already implemented

Reply:
Yes, I think that's the behavior of `npm`.
But I am always using `@latest`, so I don't think we could run into cases of downgrade

Reply:
I agree. 
But after some investigation, I don't think npm has a command built in to specify minimum versions including major versions.
Because of that, I think it doesn't make sense for us to spend much time on this, as @latest I think is enough already (even if not the best option).

Reply:
In the near term it works . . . but the longer this package exist the more we run the risk of a problem (i.e. upgrading to a breaking change).

Reply:
After looking more into the npm official docs: https://docs.npmjs.com/cli/v8/commands/npm-install#description
There is no "upgrade if needed" command. Or we make the command upgrade all the way (until some specific upper boundary) or we hard code some boundary: only upgrade until 2.x.x major (but it might have downgrades).

If we want an "upgrade if needed" command, we would have to implement it ourselves (reading and parsing the JSON). I don't think it is worth taking the time to do this work.